yt_dlp/extractor/whowatch.py

```diff
@@ -67,18 +73,28 @@ def _real_extract(self, url):
 
         self._sort_formats(formats)
 
-        uploader_id = try_get(metadata, lambda x: x['live']['user']['user_path'], compat_str)
-        uploader_id_internal = try_get(metadata, lambda x: x['live']['user']['id'], int)
+        uploader_url = try_get(metadata, lambda x: x['live']['user']['user_path'], compat_str)
+        if uploader_url:
+            uploader_url = 'https://whowatch.tv/profile/%s' % uploader_url
+        uploader_id = compat_str(try_get(metadata, lambda x: x['live']['user']['id'], int))
         uploader = try_get(metadata, lambda x: x['live']['user']['name'], compat_str)
         thumbnail = try_get(metadata, lambda x: x['live']['latest_thumbnail_url'], compat_str)
+        upload_date = try_get(metadata, lambda x: x['live']['started_at'], int)
+        if upload_date is not None:
+            upload_date = datetime.datetime.fromtimestamp(upload_date / 1000).strftime('%Y%m%d')
+        view_count = try_get(metadata, lambda x: x['live']['total_view_count'], int)
+        comment_count = try_get(metadata, lambda x: x['live']['comment_count'], int)
 
         return {
             'id': video_id,
             'title': title,
             'uploader_id': uploader_id,
-            'uploader_id_internal': uploader_id_internal,
+            'uploader_url': uploader_url,
             'uploader': uploader,
             'formats': formats,
             'thumbnail': thumbnail,
+            'upload_date': upload_date,
```

Comment:
`upload_date` will be automatically calculated from `timestamp`

```suggestion
            'timestamp': timestamp,
```